Cargo.lock

```diff
@@ -205,7 +205,7 @@ dependencies = [
 
 [[package]]
 name = "gstools-core"
-version = "0.2.0"
+version = "0.2.1"
```

Comment:
One should run at least a check build after bumping the crate version as a dirty `Cargo.lock` will trigger warnings in some build tools

Reply:
Oh... damn, this is actually changed locally on my machine, but I forgot to commit it, thanks!